config/locales/en.yml

```diff
@@ -19,7 +19,7 @@ en:
         created_at: Created
         updated_at: Updated
       inquiry:
-        is_group: Group inquiry
+        is_group: Group request
```

Comment:
@franpb14 could you please update this "inquiry -> request" change in localeapp: https://app.localeapp.com/projects/10031. There you can find the key (save into the ENV var `LOCALEAPP_API_KEY`) to later pull it via: `bundle exec localeapp pull`.